collect/central_collector.go

```diff
@@ -146,6 +146,9 @@ func (c *CentralCollector) Start() error {
 	c.Metrics.Register("kept_from_stress", "counter")
 	c.Metrics.Register("collector_send_trace", "counter")
 	c.Metrics.Register("collector_decide_trace", "counter")
+	c.Metrics.Register("decider_decided_per_second", "histogram")
```

Comment:
I'm torn between `sender` v.s. `send` in our metrics. It feels more straightforward to use `send` and `decide` but I can go either way. Just want to make sure we have a consistent pattern here

Reply:
I was distinguishing between the noun and the verb here, but I agree we should settle on a pattern. I've put it in the notes, let's decide on it together and not in this PR